src/modules/commander/HealthAndArmingChecks/checks/batteryCheck.cpp

```diff
@@ -191,17 +191,23 @@ void BatteryChecks::checkAndReport(const Context &context, Report &reporter)
 		reporter.failsafeFlags().battery_warning = worst_warning;
 	}
 
-	if (reporter.failsafeFlags().battery_warning > battery_status_s::BATTERY_WARNING_NONE
-	    && reporter.failsafeFlags().battery_warning < battery_status_s::BATTERY_WARNING_FAILED) {
+	const bool battery_warning = reporter.failsafeFlags().battery_warning > battery_status_s::BATTERY_WARNING_NONE
+				     && reporter.failsafeFlags().battery_warning < battery_status_s::BATTERY_WARNING_FAILED;
+	const bool configured_arm_threshold_in_use = !context.isArmed() && (_param_com_arm_bat_min.get() >= 0);
```

Comment:
```suggestion
	const bool configured_arm_threshold_in_use = !context.isArmed() && (_param_com_arm_bat_min.get() >= FLT_EPSILON);
```

Reply:
I included 0 so that it's possible to configure: "never disallowing arming due to low battery". Probably a detail. Does it need to be backwards compatible with `COM_ARM_BAT_MIN` specifically being set to zero to disable it?

Reply:
Okay then > -FLT_EPSILON?

Reply:
Totally, should have thought about this 🤦‍♂️

Reply:
Changed in place: https://github.com/PX4/PX4-Autopilot/compare/1eb2bec7cc5721b706e7e73341649e0f3eab6d2c..ea5641fa7f4134318da29e6a5a9ad59480671266